### PR TITLE
BGP community 2.1.0 release notes

### PR DESCRIPTION
**Pull request by jamesesilvia**
## Description
Add release notes for BGP Community plugin.

### CODE REVIEW

File: docs/plugin_bgp_community_services.md [hunk 1]
```diff
@@ -228,6 +228,14 @@ config authority bgp-community-services-profile O365 application OtherO365 bgp-c
 
 ## Release Notes
 
+### Release 2.1.0
+
+Contains scale improvements for configuration and pillar generation.
+
+#### Issues Fixed
+
+- **PLUGIN-2519** Resolve routing engine API compatability with SSR 6.1.
```
Comment (agrawalkaushik) [inline docs/plugin_bgp_community_services.md:237]: Is it 6.1.0 or some specific version of 6.1 patch that introduced this incompatibility?

Comment (laneshields-jnpr) [reply docs/plugin_bgp_community_services.md:237]: I would need help from the routing team to understand when the change in API was introduced

**Review by laneshields-jnpr — APPROVED** [review]

File: docs/plugin_bgp_community_services.md (hunk 1 above)
Comment (jamesesilvia) [reply docs/plugin_bgp_community_services.md:237]: @laneshields-jnpr do u know off hand if this was a 6.1.0 thing or 6.1.X?

Comment (jamesesilvia) [reply docs/plugin_bgp_community_services.md]: https://128technology.atlassian.net/browse/I95-46352 looks like 6.1.4. pushed the change

**Review by jeffmart-jnpr — APPROVED** [review]

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]

**Review by agrawalkaushik — APPROVED** [review]

**Pull request merged**